docs/package.json

```diff
@@ -21,12 +21,14 @@
   "dependencies": {
     "@docusaurus/core": "^2.0.0-beta.21",
     "@docusaurus/preset-classic": "^2.0.0-beta.21",
-    "@docusaurus/remark-plugin-npm2yarn": "^2.0.0-beta.21",
     "@docusaurus/theme-common": "2.0.0-beta.21",
     "@mdx-js/react": "1.6.22",
+    "@sapphire/docusaurus-plugin-npm2yarn2pnpm": "^1.1.0",
     "classnames": "^2.3.1",
+    "i": "^0.3.7",
     "mdx-mermaid": "^1.2.2",
     "mermaid": "^9.0.1",
+    "npm": "^8.13.2",
```

Comment:
I guess this was added by mistake?

Reply:
Yeah there was clearly some `npm i...` mis print along the way haha. Just up a draft PR to see if it worked on preview deploy, cause it wasn't working locally. Very much just a little test still..